spec/knapsack_pro/test_case_detectors/rspec_test_example_detector_spec.rb

```diff
@@ -11,7 +11,7 @@
 
       expect(FileUtils).to receive(:mkdir_p).with(report_dir)
 
-      expect(File).to receive(:exist?).with(report_path).and_return(true)
+      expect(File).to receive(:exist?).at_least(1).with(report_path).and_return(true)
```

Comment:
Does this change work as expected? I think it's usually used in one of those 2 ways: `at_least(:once)` / `at_least(1).time`. We might consider using one or the other per convention. Here's the link to docs: https://relishapp.com/rspec/rspec-mocks/docs/setting-constraints/receive-counts

Reply:
I've replaced it with  `at_least(:once)` to improve readability.